cmd/go-clang-generate/function.go

```diff
@@ -492,21 +602,23 @@ func generateASTFunction(f *Function) string {
 					addDefer(doCall(p.Name, "Dispose"))
 				}
 
-				// Add the return argument to the return statement
-				if p.Type.IsPrimitive {
-					addReturnItem(doCast(
-						p.Type.GoName,
-						&ast.Ident{
-							Name: p.Name,
-						},
-					))
-				} else {
-					if p.Type.GoName == "cxstring" {
-						addReturnItem(doCall(p.Name, "String"))
+				if p.Type.LengthOfSlice == "" {
```

Comment:
Must be an slice<->length pair so we can figure out how to iterate over the slice.